packages/mui-material/src/Popper/Popper.tsx

```diff
@@ -2,10 +2,22 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import PopperUnstyled, { PopperUnstyledProps } from '@mui/base/PopperUnstyled';
 import { HTMLElementType, refType } from '@mui/utils';
-import { Direction, useThemeWithoutDefault as useTheme } from '@mui/system';
+import { Direction, useThemeWithoutDefault as useTheme, SxProps } from '@mui/system';
 import useThemeProps from '../styles/useThemeProps';
+import { Theme, styled } from '../styles';
```

Comment:
```suggestion
import { styled, Theme } from '@mui/material/styles';
```

Reply:
Hi, thanks for reviewing my PR, I have made the changes according to your review. 
But if its not too much trouble, I'd like to ask why? What is the difference between using relative and absolute imports in this case?
And why do we not do the same for `useThemeProps`? Which is imported from `@mui/material/styles/useThemeProps`

Also, this change broke CI, I can't seem to figure out why, it breaks whenever it tries to build `@mui/material`.

Reply:
> But if its not too much trouble, I'd like to ask why? What is the difference between using relative and absolute imports in this case?

There is no difference. I just wanted to keep the consistency between code in the codebase. But actually, we can just do `import { Theme, styled, useThemeProps } from '../styles';`.

I will make this change